main.cpp

```diff
@@ -4,14 +4,13 @@
 int main(int argc, char *argv[]) {
 
 	// determine the startup config file...
-	const char *config_file = "default_config.cfg";
+	const char *config_file = nullptr;
```

Comment:
No, I just moved it to mainwindow.cpp:find_config_file because at that point Qt knows the executable file name and the default paths. There's (paraphrasing) `if(filename!=nullptr && filename.exists()) load(filename);` in there